src/main/java/ca/softwarespace/qiyanna/dataaggregator/repositories/sql/SQLTierRepositoryImpl.java

```diff
@@ -0,0 +1,41 @@
+package ca.softwarespace.qiyanna.dataaggregator.repositories.sql;
+
+import static ca.softwarespace.qiyanna.dataaggregator.models.generated.Tables.TIER;
+
+import ca.softwarespace.qiyanna.dataaggregator.models.dto.TierDto;
+import ca.softwarespace.qiyanna.dataaggregator.models.exceptions.RecordNotFoundException;
+import ca.softwarespace.qiyanna.dataaggregator.models.generated.tables.records.TierRecord;
+import ca.softwarespace.qiyanna.dataaggregator.repositories.sql.interfaces.SQLTierRepository;
+import org.jooq.DSLContext;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+
+@Repository
+public class SQLTierRepositoryImpl implements SQLTierRepository {
```

Comment:
Same as the other repositories, let the repository return the actual object and use services/factories to convert the object